src/main/webapp/app/shared/util/firebase/firebase-path-utils.ts

```diff
@@ -42,3 +42,20 @@ export const extractArrayPath = (valuePath: string) => {
   const firebaseArrayPath = pathParts.join('/');
   return { firebaseArrayPath, deleteIndex };
 };
+
+export enum FIREBASE_PATH_TYPE {
```

Comment:
Name is too generic. Maybe 
```suggestion
export enum FIREBASE_LIST_PATH_TYPE {
```